### DISCUSSION TITLE
Build a HMI Display for Raspberry Pi | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#840] [Q&A]
# Build a HMI Display for Raspberry Pi | Seeed Studio Wiki

Build a HMI Display for Raspberry Pi using Wio Terminal

https://wiki.seeedstudio.com/Wio-Terminal-HMI/

**Comment by SunayanPradhan** [comment]
how to use seeed studio wio terminal as a display of raspberry pi 4?

**Comment by Ryan-py** [comment]
Can you make an update for the  raspberry pi Bookworm OS.
